app-scroll-effects/app-scroll-effects.html

```diff
@@ -62,20 +84,25 @@
         this.$.backgroundRearLayer.style.opacity = 0;
       }
     }
-  };
+  },
 
   /**
    * Vertically translate the background based on a factor of the scroll position.
    */
-  scrollEffects['parallax-background'] = {
+  'parallax-background': {
+    /**
+     * @param {{scalar: string}} config
+     * @this ElementWithBackground
+     */
     setUp: function setUp(config) {
       var scalar = parseFloat(config.scalar);
+
       this._deltaBg = this.$.backgroundFrontLayer.offsetHeight - this.$.background.offsetHeight;
       if (this._deltaBg === 0) {
         if (isNaN(scalar)) {
           scalar = 0.8;
         }
-        this._deltaBg = this._dHeight * scalar;
+        this._deltaBg = this._deltaHeight * scalar;
```

Comment:
Should be `this._dHeight` instead of `this._deltaHeight` - defined in https://github.com/PolymerLabs/app-layout/blob/master/app-header/app-header.html#L419.

@blasten `_dHeight` does not exist in app-box - is that going to be an issue?